frontend/src/modules/labworkSteps/actions.ts

```diff
@@ -380,4 +382,12 @@ export const getLabworkStepSummary = (stepID: FMSId, groupBy: string, options) =
 			error: err
 		})
 	}
-}
\ No newline at end of file
+}
+
+export function setSelectedSamplesInGroups(sampleIDs: FMSId[]) {
+	return {
+		type: SELECT_SAMPLES_IN_GROUPS[1],
```

Comment:
That might actually work. I guess I was hesitant since SET_SELECTED_SAMPLES messes with `LabworkStepsState` whereas `SELECT_SAMPLES_IN_GROUPS[1]` would only mess with the `LabworkStepSummaryState`.

Reply:
isn't SELECT_SAMPLES_IN_GROUPS[1] === SET_SELECTED_SAMPLES ?

Reply:
My comment was more about simplicity and clarity than functionality ...

Reply:
> isn't SELECT_SAMPLES_IN_GROUPS[1] === SET_SELECTED_SAMPLES ?

No. It's a different value: `SAMPLES_AT_STEP:SET_SELECTED_SAMPLES_IN_GROUPS`

Reply:
I could make it an object instead of an array to make it clearer ;)